README.md

```diff
@@ -47,11 +47,11 @@ chmod 755 /usr/local/bin/stoml
 ```
 2. Install `gm`
 ```bash
-git clone https://github.com/informal/ibc-rs
-ibc-rs/scripts/gm/bin/gm install
+git clone https://github.com/informal/gm
+gm/bin/gm install
 ```
 
-Alternatively, you can create the folder `$HOME/.gm/bin` and copy the files from `scripts/gm/bin` in there.
+Alternatively, you can create the folder `$HOME/.gm/bin` and copy the files from `/bin` in there.
```

Comment:
```suggestion
Alternatively, you can create the folder `$HOME/.gm/bin` and copy the files from `bin` in there.
```

Reply:
Yes agreed. I have made the change. Thank you.